notebooks/data-sources/Bugzilla/bugzilla_EDA.ipynb

```diff
@@ -0,0 +1,3552 @@
+{
```

Comment:
<p><code>jt_dict_dj_pair_tuple</code>  should this be <code>args_tuple</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Yep, that's what I meant :)</p>